clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -1499,6 +1538,37 @@ public ClientResponse mockOffsetCommitResponse(String topic,
         );
     }
 
+    public ClientResponse mockOffsetCommitResponse(String topic,
+                                                   List<OffsetCommitResponseData.OffsetCommitResponsePartition> partitions,
+                                                   short apiKeyVersion) {
+        return mockOffsetCommitResponse(topic, partitions, apiKeyVersion, time.milliseconds(), time.milliseconds());
+    }
+
+    public ClientResponse mockOffsetCommitResponse(String topic,
+                                                   List<OffsetCommitResponseData.OffsetCommitResponsePartition> partitions,
+                                                   short apiKeyVersion,
+                                                   long createdTimeMs,
+                                                   long receivedTimeMs) {
+        OffsetCommitResponseData responseData = new OffsetCommitResponseData()
+                .setTopics(Collections.singletonList(
+                        new OffsetCommitResponseData.OffsetCommitResponseTopic()
+                                .setName(topic)
+                                .setPartitions(partitions)));
+        OffsetCommitResponse response = mock(OffsetCommitResponse.class);
+        when(response.data()).thenReturn(responseData);
+        return new ClientResponse(
+                new RequestHeader(ApiKeys.OFFSET_COMMIT, apiKeyVersion, "", 1),
+                null,
+                "-1",
+                createdTimeMs,
+                receivedTimeMs,
+                false,
+                null,
+                null,
+                new OffsetCommitResponse(responseData)
+        );
+    }
+
     public ClientResponse mockOffsetCommitResponseDisconnected(String topic, int partition,
```

Comment:
same for this one, not introduced here but I would expect all this mock helpers to be private to this test

Reply:
Thanks for point out.
I will change these helper methods to private in this PR 😺